Sources/Brave/Frontend/Shields/SubmitReportSuccessView.swift

```diff
@@ -0,0 +1,71 @@
+// Copyright 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import Strings
+import BraveUI
+import DesignSystem
+import SnapKit
+
+struct SubmitReportSuccessView: View {
+  var body: some View {
+    VStack(alignment: .leading) {
+      Text(Strings.Shields.siteReportedTitle)
+        .multilineTextAlignment(.leading)
+        .lineLimit(1)
+        .font(.title)
+        .foregroundStyle(Color(braveSystemName: .textTertiary))
+      Text(Strings.Shields.siteReportedBody)
+        .multilineTextAlignment(.leading)
+        .lineLimit(3)
+        .foregroundStyle(Color(braveSystemName: .textTertiary))
+    }
+    .padding()
+  }
+}
+
+#if swift(>=5.9)
+#Preview {
+  SubmitReportSuccessView()
+}
+#endif
+
+class SubmitReportSuccessViewController: UIViewController, PopoverContentComponent {
```

Comment:
If this is going to stay a popover than consider using the newer overload in `PopoverController`: `init(content:autoLayoutConfiguration:)` which takes a SwiftUI View